### ISSUE TITLE
Support for VisualStudio 2022

### ISSUE THREAD

**Issue by ornic** [league1991/CodeAtlasVsix#19]
Subj :)

**Comment by jbienzms** [comment]
Yes, please add support for Visual Studio 2022. Unfortunately, this will not be a simple .vsixmanifest change like in previous versions. 😔 Here are the instructions:

https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022

**Comment by fs-srogers** [comment]
Using MacOS 12.4 Monterey vscode install fails:
json not found in zip

**Comment by Taz5150** [comment]
Bump on this.

**Comment by league1991** [comment]
Sorry for late... I will try that when I have time. But I am not sure how long it will take...

**Comment by lukas-hofstaetter** [comment]
Hey @league1991 ,
I just discovered your outdated plugin. Is it right, that I can generate CallGrapgs for C and C++? 

Have you thought about opening a sponsor/donate page for this? Or is it just a time bottle neck for you?

**Comment by MaksimMelan** [comment]
I saw the addition and I liked it so much. I would like to install it, but it does not fit VS 2022

**Comment by league1991** [comment]
@eDeviser Sorry for late. Yes, you can generate call graph for C/C++. The current VS plugin only supports VS2019 and previous versions.

Before you use it, you'd better watch [this](https://www.youtube.com/watch?v=FIdT3rCPpo0) video.

**Comment by league1991** [comment]
@MaksimMelan Sorry, but I am afraid I don't have enough time to work on VS2022 right now. The latest version is still VS2019.

**Comment by denyeo** [comment]
> Yes, please add support for Visual Studio 2022. Unfortunately, this will not be a simple .vsixmanifest change like in previous versions. 😔 Here are the instructions:
> https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022

Wow, that is annoying. Does anyone know the minimal changes needed to get the existing code working in VS 2022 alone? Forget the modernization and shared project changes - do any package references need to be updated?

This looks like a really useful extension. I wish I could help upgrade it, but I'm new to Visual Studio.
